lib/Conversion/TorchToArith/TorchToArith.cpp

```diff
@@ -401,9 +402,15 @@ class ConvertTorchToArith
     patterns.add<
         ConvertAtenIntComparisonOp<AtenGtIntOp, arith::CmpIPredicate::sgt>>(
         typeConverter, context);
+    patterns.add<
+        ConvertAtenIntComparisonOp<AtenLtIntOp, arith::CmpIPredicate::slt>>(
+        typeConverter, context);
     patterns.add<
         ConvertAtenIntComparisonOp<AtenGeIntOp, arith::CmpIPredicate::sge>>(
         typeConverter, context);
+    patterns.add<
```

Comment:
It's at least consistent with the other patterns, but I do wonder if there is a need to differentiate between signed/unsigned int types.

Looks good to me.